src/ucp/rma/amo_send.c

```diff
@@ -107,6 +107,8 @@ ucs_status_ptr_t ucp_atomic_fetch_nb(ucp_ep_h ep, ucp_atomic_fetch_op_t opcode,
     ucs_status_t status;
     ucp_request_t *req;
 
+    UCP_CONTEXT_CHECK_FEATURE_FLAGS(ep->worker->context, UCP_FEATURE_AMO,
```

Comment:
Do you mean according to `op_size`? Then we should modify existing `UCP_AMO_CHECK_PARAM` for exact values instead of `((_remote_addr) % (_size)) != 0`